ibm/config.go

```diff
@@ -1307,6 +1318,28 @@ func (c *Config) ClientSession() (interface{}, error) {
 	}
 	session.appidAPI = appIDClient
 
+	// Construct an "options" struct for creating Context Based Restrictions  service client.
+	cbrURL := contextbasedrestrictionsv1.DefaultServiceURL
```

Comment:
visibility and filemap checks are missing.. add something similar to [this](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/config.go#L2447)
```
if c.Visibility == "private" || c.Visibility == "public-and-private" {
		cbrURL = contructEndpoint("private.cbr", cloudEndpoint) //add this only if private endpoints are supported..
		session.contextBasedRestrictionsClientErr = fmt.Errorf("Context Based Restrictions Service API doesnot support private endpoints") //return this error if private endpoints are not supported
	}
	if fileMap != nil && c.Visibility != "public-and-private" {
		cbrURL = fileFallBack(fileMap, c.Visibility, "IBMCLOUD_CONTEXT_BASED_RESTRICTIONS_ENDPOINT", c.Region, cbrURL)
	}
```

Reply:
Added above code.
we don't support Private endpoint